plugins/LilyPad/LilyPad.rc

```diff
@@ -450,8 +450,8 @@ BEGIN
     CONTROL         "Raw input",IDC_KB_RAW,"Button",BS_AUTORADIOBUTTON,22,40,112,10
     CONTROL         "DirectInput",IDC_KB_DI,"Button",BS_AUTORADIOBUTTON,22,52,112,10
     GROUPBOX        "Game Device APIs",IDC_STATIC,16,70,191,62
-    CONTROL         "DirectInput",IDC_G_DI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,82,65,10
-    CONTROL         "XInput",IDC_G_XI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,94,125,10
+    CONTROL         "DirectInput (Legacy)",IDC_G_DI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,82,100,10
+    CONTROL         "XInput (For modern controllers)",IDC_G_XI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,94,125,10
```

Comment:
Maybe just `XInput (Modern)` ?

Reply:
Yeah, good idea, just had that there to be obvious it's for controllers.